packages/react/src/dialog/popup/useDialogPopup.tsx

```diff
@@ -27,9 +36,16 @@ export function useDialogPopup(parameters: useDialogPopup.Parameters): useDialog
     titleElementId,
   } = parameters;
 
+  const handleFloatingUIOpenChange = React.useCallback(
+    (isOpen: boolean, event: Event | undefined, reason: FloatingUIOpenChangeReason | undefined) => {
+      setOpen(isOpen, event, translateOpenChangeReason(reason));
+    },
+    [setOpen],
+  );
```

Comment:
`useEventCallback`

Reply:
This depends on `setOpen` only, which is stable.

Reply:
Yeah, but you need to specify the deps needlessly, plus for consistency it's good to wrap event handlers in it since it's used elsewhere